lib/bookstore-scrapper/index.js

```diff
@@ -0,0 +1,307 @@
+/* Instructions:
+
+1. Install Deno CLI: https://deno.land/manual/getting_started/installation
+2. Run "deno run --allow-write --allow-net index.js"
+
+*/
+
+// Variables used to build links that connect to campus store
+let storeDomain = 'https://campusstore.mcmaster.ca';
+let storeBookPath = '/cgi-mcm/ws';
+let storeCourseMaterialPath = '/txhome.pl?wsgm=coursematerial';
+let storeFindBookPath = '/txsub.pl?';
+let storeImagePath = "/getTradeImage.pl?isbn=";
+let storeTermG1Search = 'wsTERMG1=';
+let storeTermDescSearch = '&wsTERMDESC1=';
+let storeDeptG1Search = '&wsDEPTG1=';
+let storeDeptDescSearch = '&wsDEPTDESC1=';
+let storeCourseG1Search = '&wsCOURSEG1=';
+let storeSectionG1Search = '&wsSECTIONG1=';
+let storeProgramG1Search = '&programG1=';
+let storeFindBookPrefix = '&crit_cnt=1';
+
+// Here is the path to generate the link to the book image
+let bookImagePath = `${storeDomain}${storeBookPath}${storeImagePath}`
+
+// Parse variables from page to use eval
+let startStoreVariablesScrape = "/// Program/Terms";
+let endStoreVariablesScrape = "</script>";
+let varDefine = 'var';
+let arrayDefine = "new Array(";
+let arrayEnd = ");";
+
+// Values to parse options from pages
+let startParseProgram = "<option value=\"";
+let endParseSlash = "\"";
+let parseISBN = '?isbn=';
+let parseBookName = 'alt="'
+
+// Values to parse price from book page
+let startParsePrice = "value=\"";
+let midParsePrice = "\" onclick=\"toggle";
+let endParsePrice = "new";
+let quoteParsePrice = "\'"
+let priceNotAvailable = 'Not Available';
+
+// Get price from book page
+function getBookPrice(bookHTML, ISBN) {
```

Comment:
Note that getBookPrice was the only function added from previous version of script.